app/gui/src/project-view/components/GraphEditor.vue

```diff
@@ -715,7 +749,9 @@ const contextMenuActions: ActionName[] = [
         <CodeEditor ref="codeEditor" />
       </BottomPanel>
     </div>
-    <RightDockPanel ref="docPanel" v-model:displayedDocs="displayedDocs" :aiMode="aiMode" />
+
+    <slot name="rightDock" />
```

Comment:
Here we put a slot with the right dock panel in react, it's a straightforward implementation to make the provide/inject work for the `component help` component.